lib/http/apps/wallet.js

```diff
@@ -15,37 +15,49 @@ const password = process.env.PASSWORD || "coinsence";
  * @apiGroup Wallet
  * @apiVersion 1.0.0
  * 
- * @apiParam {Number} accountId Accounts unique ID.
+ * @apiParam {String[]} accountsIds Accounts unique ID.
  * 
  * @apiUse PostWalletSuccess
  * 
- * @apiUse MissingAccountIdError
- * @apiUse ExistantWalletError
- * @apiUse InternalServerError
+ * @apiUse MissingAccountsIdsError
  */
 app.post(`/`, (req, res) => {
-  createEncryptedWallet(res.locals.accountId, password).then(wallet => {
-    if (res.locals.coinsenceSigner) {
-      res.locals.coinsenceSigner.sendTransaction({
-        to: wallet.address,
-        value: ethers.utils.parseEther('0.1')
+  const accountsIds = req.body.accountsIds;
+  let result = [];
+
+  if (!Array.isArray(accountsIds)) {
+    res.status(400).json({error: 'accountIds is missing'});
+  } else {
+    accountsIds.forEach((accountId) => { 
+      createEncryptedWallet(accountId, password).then(wallet => {
```

Comment:
I think we need to use `await` here - otherwise the response happens too early. 

or we refactor it to something like this: 
```js
resultPromises = accountsIds.map((accountId) => { 
  return createEncryptedWallet(accountId, password).then(wallet => { 
    // ...
    return { accountId: accountId, address: wallet.address };
  });
});
Promise.all(resultPromises).then(results => {
  res.status(201).json(results);
});
```

Reply:
so we wait until every wallet is created and the object is returned and then we return the whole thing.